osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -222,6 +227,9 @@ public MenuItem[] ContextMenuItems
                 if (Item?.State.Value == CarouselItemState.NotSelected)
                     items.Add(new OsuMenuItem("Expand", MenuItemType.Highlighted, () => Item.State.Value = CarouselItemState.Selected));
 
+                if (mainMenuItems != null)
```

Comment:
I found a simplified way to do this:
```cs
items.AddRange(mainMenuItems ?? Array.Empty<MenuItem>());
```
What do you think?

Reply:
nope